### PR TITLE
Feature/list txs by generation

### PR DESCRIPTION
**Pull request by shekhar-shubhendu**

### CODE REVIEW

**Comment by shekhar-shubhendu** [comment]
closes #318

File: src/store/modules/transactions.js [hunk 1]
```diff
@@ -32,9 +31,15 @@ export default {
       if (state.transactions[hash]) return state.transactions[hash]
 
       const transaction = await epoch.tx(hash, true)
+      console.log(transaction)
       commit('setTransaction', transaction)
 
       return transaction
+    },
+    async getTxByGeneration ({ state, commit, rootGetters: { epoch } }, { start, end }) {
+      const listTxRequest = await fetch(`${process.env.VUE_APP_EPOCH_URL}transactions/interval/${start}/${end}`)
```
Comment (ror-shubham) [inline src/store/modules/transactions.js]: This should be
```const listTxRequest = await fetch(`${process.env.VUE_APP_EPOCH_URL}middleware/transactions/interval/${start}/${end}`)```

**Comment by shekhar-shubhendu** [comment]
@ror-shubham As advised, rebased the branch to `develop-middleware` and added the load more button.

File: src/app.vue [hunk 2]
```diff
@@ -139,11 +139,16 @@ export default {
...
       })[this.$route.name]
-    }
+    },
+    ...mapState('blocks', {
+      height: ({ height }) => height })
   },
   methods: {
     toggleMenu: function () {
       this.isOpened = !this.isOpened
+    },
```
Comment (ror-shubham) [inline src/app.vue]: This can be a computed prop

File: src/app.vue [hunk 3]
```diff
@@ -139,11 +139,16 @@ export default {
         'GenerationList': 'Generations',
         'Transaction': 'Transactions'
       })[this.$route.name]
-    }
+    },
+    ...mapState('blocks', {
```
Comment (ror-shubham) [inline src/app.vue]: Can be replaced by 
```
...mapState('blocks', ['height']),
```

**Comment by ror-shubham** [comment]
@shekhar-shubhendu I did some minor changes regarding the above comments. Please have a review of the commit, and if you find them acceptable, please squash and merge.

**Pull request merged**